src/games/supported/Othello.cpp

```diff
@@ -42,18 +42,19 @@ void OthelloSettings::step(const System& system) {
   m_reward = score - m_score;
   m_score = score;
 
-  // On screen cursor flashes every 4 frames when player input is accepted.
-  if (readRam(&system, 0xe8) == 0) {
-    ++m_cursor_inactive;
+  // Player indicator is 0xff if white's turn, 0x01 if black's turn and 0x00
+  // if game is over. Also is 0x00 in other situations, but only temporarily
```

Comment:
Sorry, one more nitpick: "if <ins>the</ins> game is over. Also<ins>, it</ins>is 0x00 ...", and a full stop at the end please?

Reply:
Fixed